tests/snippets/math_module.py

```diff
@@ -0,0 +1,120 @@
+import math
```

Comment:
The content of this file looks good. We are putting any stdlib test code into `stdlib_<name>.py` test file. So putting this test into `stdlib_math.py` will be better. https://github.com/RustPython/RustPython/blob/master/tests/snippets/stdlib_math.py

Reply:
@adolfogonzales3 Are you continuing with the PR?